specs/paymentrequest.html

```diff
@@ -1105,7 +1114,7 @@ <h2 id="state-transitions">State transitions</h2>
             The <a>user agent</a> user interface should ensure that this never occurs.
           </li>
           <li>
-            If the <code>requestShipping</code> value of <em>request</em>@[[\options]]
+            If <code>requestedData</code> contains <code>shipping-address</code> in <em>request</em>@[[\options]]
             is <code>true</code>, then if the <code>shippingAddress</code> attribute of <em>request</em>
             is <code>null</code> or if the <code>shippingOption</code> attribute of <em>request</em>
```

Comment:
Line 1118 should probably change from
`is <code>true</code>, then if the <code>shippingAddress</code> attribute of <em>request</em>`
to
`then if the <code>shippingAddress</code> attribute of <em>request</em>`
or something like that...